src/pages/ResearcherProfile.js

```diff
@@ -13,6 +13,8 @@ import { Notification } from '../components/Notification';
 import * as ld from 'lodash';
 import { USER_ROLES, setUserRoleStatuses } from '../libs/utils';
 
+const UnitedStatesOrEmpty = ["united states", "us", "united states of america", "usa", ''];
```

Comment:
A less brittle way of doing this is to change the select to use a list of standardized country codes: https://www.npmjs.com/package/country-list
That way, you can inspect what the user selects and know for sure if they chose US or not.
This approach of checking on strings can be very brittle, i.e., it will miss variants and misspellings like "U nited States" or "US of A", etc.